### PR TITLE
Add camera_rate param.

### PR DESCRIPTION
**Pull request by jakeware**
Add a ros param that sets a timer frequency that teensy uses to trigger the camera at a given frame rate.

### CODE REVIEW

File: ros/src/svis_ros_nodelet.cc [hunk 1]
```diff
@@ -325,8 +335,13 @@ class SVISNodelet : public nodelet::Nodelet {
 
   void SendSetup() {
     std::vector<char> buf(64, 0);
+    // header
     buf[0] = 0xAB;
     buf[1] = 0;
```
Comment (wngreene) [inline ros/src/svis_ros_nodelet.cc]: Magic numbers?

File: ros/CMakeLists.txt [hunk 2]
```diff
@@ -25,6 +25,7 @@ set(REQ_CATKIN_PKGS
   std_msgs
   sensor_msgs
   message_generation
+  fla_utils
```
Comment (wngreene) [inline ros/CMakeLists.txt]: Given that this is supposed to be a general purpose tool, do you really need the dependency on `fla_utils`? If you just need `SafeGetParam`, I would just copy it into an `svis` specific utils file.

File: ros/src/svis_ros_nodelet.cc (hunk 1 above)
Comment (wngreene) [inline ros/src/svis_ros_nodelet.cc]: I would make the first two bytes constants.

Comment (jakeware) [inline ros/src/svis_ros_nodelet.cc]: Those are just header bytes to indicate what type of packet is coming across.  I intend the first one (0xAB) to never change and the second byte to indicate packet type.  I put a checksum at the end of the packets going from the teensy to ros, and will do the same for this once I get around to it.  Does this match what you are suggesting?

File: ros/CMakeLists.txt (hunk 2 above)
Comment (jakeware) [inline ros/CMakeLists.txt]: A good suggestion.

**Comment by jakeware** [comment]
Great.  Definitely interested to hear your thoughts on the project as a whole.  I know there are a lot of rough edges and barely implemented functionality.  One foot in front of the other, though.

File: ros/src/svis_ros_nodelet.cc (hunk 1 above)
Comment (wngreene) [inline ros/src/svis_ros_nodelet.cc]: Checksum is definitely a good idea, but I was thinking something more like:
```c++
 public:
  constexpr uint8_t PACKET_HEADER = 0xAB;
  enum PacketType {
    Packet0 = 0,
    Packet1 = 1
  }; 
```

Comment (jakeware) [inline ros/src/svis_ros_nodelet.cc]: Ah, an enum is a good idea.

**Pull request merged**